### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by jor-el-swe**

### CODE REVIEW

**Comment by Exoduz85** [comment]
WHAT THE FUCK IS THIS???!!! you son of a... :D

**Comment by Exoduz85** [comment]
fuck yeah, i fucking hate that fucking folder!!!!
let's git rid of it 🗡️

File: .gitignore [hunk 1]
```diff
@@ -2,13 +2,27 @@
 #
 # Get latest from https://github.com/github/gitignore/blob/master/Unity.gitignore
 #
-/[Ll]ibrary/
-/[Tt]emp/
```
Comment (Alex-Rynkowski) [inline .gitignore:6]: What is dis

**Review by Alex-Rynkowski — COMMENTED** [review]
Nice you added .idea!

**Pull request merged**